### ISSUE TITLE
Possible memory leak

### ISSUE THREAD

**Issue by SrBedrock** [mfnalex/ChestSort#185]
**Used ChestSort version**
Tell us your ChestSort version
```shel
[13:16:31 INFO]: ChestSort version 14.0.0
[13:16:31 INFO]: Allows automatic chest sorting!
[13:16:31 INFO]: Website: https://www.chestsort.de
[13:16:31 INFO]: Authors: mfnalex and JEFF Media GbR
```

**Used Spigot version**
```shel
[13:16:50 INFO]: Current: git-Purpur-2153 (MC: 1.20.4)*
Previous: git-Purpur-2152 (MC: 1.20.4)
* You are running the latest version
```

**Describe the bug**

**Additional context**
![image](https://github.com/mfnalex/ChestSort/assets/51332006/e52024b5-a3bb-4364-8aeb-f1decaec367f)

![image](https://github.com/mfnalex/ChestSort/assets/51332006/2a6580e9-ed62-4c4d-81f6-345797be4401)

![image](https://github.com/mfnalex/ChestSort/assets/51332006/f55b0998-661e-47b5-9425-5e3a533bd63c)

![image](https://github.com/mfnalex/ChestSort/assets/51332006/06fbbdcf-b972-4c2c-9196-86bf2ae730de)

**Comment by mfnalex** [comment]
Thanks for reporting & sorry for the late reply. Will be fixed in the next update.

**Comment by mfnalex** [comment]
Fixed in 14.0.6: https://www.spigotmc.org/resources/chestsort-api.59773/update?update=547687
